hack/tools/release/internal/update_providers/provider_issues.go

```diff
@@ -249,42 +274,63 @@ func continueOrAbort() {
 }
 
 // getReleaseDetails returns the release details from the environment variables.
-func getReleaseDetails() releaseDetails {
+func (m *ProviderIssues) getReleaseDetails() releaseDetails {
```

Comment:
Just return an additional error parameter here instead of adding all the exiter complexity :)

Reply:
😬 Simpler, yes. Will update.

Reply:
Addressed via 4132310f013298e5551a1249c4718ca1a7337e83